static/scss/main.scss

```diff
@@ -22,6 +22,8 @@ $xlarge:      new-breakpoint(max-width 1200px, 12);
 $dark-blue:       darken(#0066CC, 20%);
 $medium-blue:     darken(#0066CC, 5%);
 $light-blue:      #0066CC;
+$reward:          #2da52d;
```

Comment:
As cute as this is, I would prefer we stick to variable convention with $green and $red and just leave reward/punishment as comments.